deploy/helm/grafana-operator/values.yaml

```diff
@@ -77,12 +86,17 @@ podAnnotations: {}
 # -- pod security context
 podSecurityContext: {}
 
-# -- grafana operator container security context
+# https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
 securityContext:
+  # -- grafana operator does not require default capabilities
   capabilities:
     drop:
       - ALL
+  # -- Disables privilege escalation inside the container
+  allowPrivilegeEscalation: false
+  # -- Allow writing to filesystem
```

Comment:
The comment here contradicts the actual behaviour. I would name it like `# -- Whether to enforce read-only filesystem`.